### PR TITLE
Add papernodes as easy

### PR DESCRIPTION
**Pull request by 3213141234**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -14946,6 +14946,17 @@
         ]
     },
 
+    {
+        "name": "PaperNodes",
+        "url": "https://papernodes.com",
+        "difficulty": "easy",
+        "notes": "Hover over More Pages then click on login, then go to account settings scroll down and u see Delete my account then just press the Delete Account button then enter your password and all the data asociated to your account will be deleted in 48 hours.",
```
Comment (tupaschoal) [inline _data/sites.json:14953]:
```suggestion
        "notes": "Hover over 'More Pages' then click on 'login' ⮕'account settings' and scroll down to see 'Delete my account' and press it. After confirming your password, all the data asociated to your account will be deleted in 48 hours.",
```

**Review by tupaschoal — COMMENTED** [review]
Please move the entry on the file so it remains sorted by name, as stated on the failing check.

**Comment by 3213141234** [comment]
Ok

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**